src/main/resources/db/migration/postgres/V1_333_2__insert_security_groups.sql

```diff
@@ -0,0 +1,1054 @@
+INSERT INTO darts.security_group(
```

Comment:
Should we use current_timestamp for the date rather than hard coding?

Reply:
we wanted to make it obvious that the data was taken from Production and done via migration pre-go-live. current_timestamp would be more applicable after-go-live i think. i.e. it is really obvious that the data has been manually populated as the date will be exactly 1st January 00:00.